packages/openapi-v3/test/unit/controller-spec/controller-decorators/request-body-decorators/short-cut.test.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/openapi-v3
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {expect} from '@loopback/testlab';
+import {getControllerSpec, requestBody, post} from '../../../../../';
+
+describe('requestBody decorator - shortcuts', () => {
+  context('array', () => {
+    it('', () => {
+      const description = 'an array of names';
+      class MyController {
+        @post('/greeting')
+        greet(
+          @requestBody.array(
+            {type: 'string'},
+            {description: description, required: false},
+          )
+          name: string[],
+        ) {}
+      }
+
+      const actualSpec = getControllerSpec(MyController);
+      const expectedContent = {
+        'application/json': {
+          schema: {
+            type: 'array',
+            items: {type: 'string'},
+          },
+        },
+      };
+
+      expect(
+        actualSpec.paths['/greeting']['post'].requestBody.description,
```

Comment:
The snippet `actualSpec.paths['/greeting']['post'].requestBody` is a long one and repeated three times. Please extract it into a variable.

Even better, use `properties` assertion.

```ts
expect(actualSpec.paths['/greeting']['post'].requestBody).to.have.properties({
  description: '...',
  required: false,
  content: expectedContent,
});
```